config/components/kueueviz/backend-deployment.yaml

```diff
@@ -20,3 +20,10 @@ spec:
           imagePullPolicy: IfNotPresent
           ports:
             - containerPort: 8080
+          resources:
+            limits:
+              cpu: 500m
+              memory: 512Mi
+            requests:
+              cpu: 500m
+              memory: 512Mi
```

Comment:
I think this looks a bit of over-allocation since this is just web backend server.
However, I'm not sure what exactly the optimal resources.

Reply:
Yes, me too. With this PR my main intention is to avoid releasing without any requests and limits. 

Maybe the backend is using some caching and then it might need more mem. In any case, I'm happy to update according to the expertise of @akram or @kannon92 .

Reply:
SGTM

Reply:
It would probably be a good idea to make this configurable for Helm.

Reply:
I don't think this is a blocker for release but I don't yet run/test kueue-viz very often so I don't have a good feeling for what values actually make sense.